sources/packages/web/src/services/InstituteService.ts

```diff
@@ -3,6 +3,11 @@ export interface Institute {
   code?: string;
 }
 
+export interface EducationProgram {
+  name: string;
+  code?: string;
```

Comment:
Is there a reason to assume that this would be nullable?

Reply:
For now this is placeholder only, we will revisit this after detailed API and data modelling.